opal_common/git/repo_cloner.py

```diff
@@ -59,34 +59,69 @@ def __init__(self, base_clone_path: str, clone_subdirectory_prefix: str):
 
         self._base_clone_path = os.path.expanduser(base_clone_path)
         self._clone_subdirectory_prefix = clone_subdirectory_prefix
+        self._use_fixed_path = use_fixed_path
 
-    def get_clone_subdirectories(self) -> Generator[str, None, None]:
+    def _get_random_clone_subdirectories(self) -> Generator[str, None, None]:
```

Comment:
why the prefix "_"?
also maybe better called:
get_randomized_clone_subdirectories (so it's clearer its not a new random)